playbooks/common/openshift-cluster/upgrades/pre/verify_health_checks.yml

```diff
@@ -11,3 +11,6 @@
       checks:
       - disk_availability
       - memory_availability
+      - docker_image_availability
+      - etcd_imagedata_size
```

Comment:
Should we block an upgrade if "etcd has too much image data"? That may not be a good idea.